sktime/transformations/series/simple_fourier.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+"""Simple Fourier transform for time series, interfacing numpy rfft."""
+
+__author__ = ["blazingbhavneek"]
+
+
+import numpy as np
+import pandas as pd
+from numpy.fft import rfft, rfftfreq
+
+from sktime.transformations.base import BaseTransformer
+
+
+class SimpleFourier(BaseTransformer):
+    r"""Simple Fourier transform for time series.
+
+    Fourier Transform is used to test data for periodicity.
+    Non-periodic/Noisy/Random data will have limited and
+    random amplitudes for frequencies, whereas periodic data
+    will have exceptional amplitudes for certain frequencies.
+
+    The implementation is based on the fast fourier transform from numpy.fft
+
+    Examples
+    --------
+    >>> from sktime.transformations.series.simple_fourier import SimpleFourier
+    >>> from sktime.datasets import load_airline
+    >>> X = load_airline()
+    >>> transformer = SimpleFourier()
+    >>> X_ft = transformer._transform(X)
+    """
+
+    _tags = {
+        "scitype:transform-input": "Series",
+        "scitype:transform-output": "Series",
+        "scitype:instancewise": True,
+        "scitype:transform-labels": "None",
+        "X_inner_mtype": "np.ndarray",
+        "y_inner_mtype": "None",
+        "univariate-only": False,
+        "requires_y": False,
+        "fit_is_empty": True,
+        "capability:inverse_transform": False,
+        "capability:unequal_length": True,
+        "handles-missing-data": False,
+    }
+
+    def __init__(self):
+        super(SimpleFourier, self).__init__()
+
+    def _transform(self, X):
```

Comment:
`transform()` method of [BaseTransformer](https://github.com/sktime/sktime/blob/main/sktime/transformations/base.py#L504) calls `._transform()` with a y parameter as well. In your implementation, there's no y parameter and hence the error. 

I suggest passing y even if the operation is solely on the predictors.

Reply:
Done!